include/boost/random/detail/traits.hpp

```diff
@@ -0,0 +1,107 @@
+/* boost random/detail/seed.hpp header file
```

Comment:
> In include/boost/random/detail/traits.hpp 
> <https://github.com/boostorg/random/pull/16#discussion_r26088928>:
>
> > @@ -0,0 +1,107 @@
> > +/* boost random/detail/seed.hpp header file
>
> Should be traits.hpp. Also, is this a public header or not? If it's 
> not public, it should not be included in the documentation and should 
> not be visible to users. If it is public, it shouldn't be under detail/.
>
OK, moved to boost/random.  This is pretty much a "documented detail", I 
wouldn't expect anyone to need to specialize these traits, but you never 
know.  Better documented than not IMO.